tests/unit/simple_tests.js

```diff
@@ -90,17 +90,25 @@ Object.keys(env).forEach(key => {
     describe(`Unit tests with mockup ${key} server`, () => {
         let server;
         let client;
+        let sockets;
 
         beforeEach('start server', done => {
-            server = e.s(handler).listen(9000, done).on('error', done);
             client = e.c;
+            sockets = [];
+            server = e.s(handler)
```

Comment:
@nicolas2bert adding the sockets destroy fixed it, but let me know if you think I should do something else

Reply:
Nice catch!
We can force closing the open connections left by destroying the client HTTP(s) agent.

```
afterEach('stop server and close connections', () => { 
    server.close(); 
    e.c.agent.destroy(); 
});
```